chart/values.yaml

```diff
@@ -468,6 +468,9 @@ webserver:
 
 # Flower settings
 flower:
+  # Enable flower.
```

Comment:
i think you can chop this line because 472 already does a better job of explaining